lib/commands/cryptographic-identities.ts

```diff
@@ -605,12 +605,12 @@ export class ImportCryptographicIdentity implements ICommand {
 					this.$logger.error(err.message + os.EOL + "Verify that you have provided the correct file and password and try again.");
 				}
 
-				if (noErrorOccurred) {
+				if (noErrorOccurred || importType === CryptographicIdentityConstants.X509CERTIFICATE) {
 					break;
 				}
 			}
 
-			if (!noErrorOccurred) {
+			if (!noErrorOccurred && importType === CryptographicIdentityConstants.PKCS12CERTIFICATE) {
```

Comment:
more confusing - if there's an error, but the certificate is not PKCS12, we'll not fail, again we'll show that the certificate is imported successfully.